src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setFigureType("circle");
+        setColor(color);
+        calculateArea();
+    }
```

Comment:
See my comments to the `Figure` class, firstly change them, and then you will need to change all the inheritors.